tables_list.csv

```diff
@@ -0,0 +1,96 @@
+Name,Description
+agentservices_tasky_agent_states_v2,"This table contains the Tasky Agent States"

```

Comment:
So we need to commit the CSV as well here? I thought google sheet would remain the source of truth & xml will be updated in case of any changes there so committing this seems redundant to me.

Reply:
No CSV or XML files are to be commited. They are here so that Yogesh can directly refer them.